lib/jobba.rb

```diff
@@ -38,4 +38,12 @@ def self.redis
     )
   end
 
+  # Clears the whole shebang!  USE WITH CARE!
+  def self.clear_all_jobba_data!
+    keys = Jobba.redis.keys("*")
```

Comment:
Pretty sure `keys("*")` explodes in prod, but if it's intended to be used only in tests, it's fine.